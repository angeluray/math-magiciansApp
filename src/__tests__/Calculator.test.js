import React from 'react';
import renderer from 'react-test-renderer';
import MyCalculator from '../components/Calculator';

describe('Check if Calculator is rendering', () => {
  test('Renders correctly', () => {
    const tree = renderer.create(<MyCalculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
