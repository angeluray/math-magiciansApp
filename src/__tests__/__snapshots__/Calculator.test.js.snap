// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if Calculator is rendering Renders correctly 1`] = `
<div
  className="calculatorContainer"
>
  <div
    className="resultScreen"
  />
   
  <div
    className="calculatorButtons"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`Check if Calculator is rendering Renders correctly: 600 1`] = `
<div
  className="calculatorContainer"
>
  <div
    className="resultScreen"
  />
   
  <div
    className="calculatorButtons"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
